.pipelines/templates/build-dummy-package-for-nuget.yml

```diff
@@ -0,0 +1,33 @@
+# config to pass through parameters and insert the options required to build a dummy nuget package with no operators
+# via build-package-for-nuget.yml
+parameters:
+- name: DoCompliance
+  displayName: Run Compliance Tasks?
+  type: boolean
+  default: true
+
+- name: DoEsrp
+  displayName: Run code sign tasks? Must be true if you are doing an OnnxRuntime extensions release.
+  type: boolean
+  default: true
```

Comment:
do we have to run compliance or code sign for the dummy package?

Reply:
I assume so as we need to publish it. Technically all compliance checks are done for the real package so don't need to be repeated, but it's easier to not have to explain why we skip them here if someone asks. It's a one-off build and publish so any cost from doing so is minimal.